micrometer-tracing-bridges/micrometer-tracing-bridge-otel/src/main/java/io/micrometer/tracing/otel/bridge/OtelSpan.java

```diff
@@ -57,6 +57,9 @@ static io.opentelemetry.api.trace.Span toOtel(Span span) {
     }
 
     static Span fromOtel(io.opentelemetry.api.trace.Span span) {
+        if (span instanceof OtelSpan) {
+            return (OtelSpan) span;
```

Comment:
Does this cast have any effect when the return type is `Span` anyways?

Reply:
Nope